apps/web/src/components/Calendar/header/date/calendar-header-date-chevrons.tsx

```diff
@@ -47,7 +47,7 @@ export default function CalendarHeaderDateChevrons () {
       </Button>
 
       <span className='min-w-[140px] text-center font-medium'>
-        {DateTime.fromJSDate(date).toFormat('MMMM d, yyyy')}
+        {DateTime.fromJSDate(date).toFormat('MMMM yyyy')}
```

Comment:
flag for the future - cover the date time format to the local format used by people around the world in their country